app/assets/images/payment_icons/ask.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 67 23" width="38" height="24" aria-labelledby="pi-ask"><title id="pi-ask">ASK</title><path d="M17.32 17.5H7.57L5.71 22H.73l9.36-21h4.8l9.39 21h-5.1l-1.86-4.5zm-1.53-3.69l-3.33-8.04-3.33 8.04h6.66zm17.929 8.55c-1.66 0-3.27-.22-4.83-.66-1.54-.46-2.78-1.05-3.72-1.77l1.65-3.66c.9.66 1.97 1.19 3.21 1.59 1.24.4 2.48.6 3.72.6 1.38 0 2.4-.2 3.06-.6.66-.42.99-.97.99-1.65 0-.5-.2-.91-.6-1.23-.38-.34-.88-.61-1.5-.81-.6-.2-1.42-.42-2.46-.66-1.6-.38-2.91-.76-3.93-1.14-1.02-.38-1.9-.99-2.64-1.83-.72-.84-1.08-1.96-1.08-3.36 0-1.22.33-2.32.99-3.3.66-1 1.65-1.79 2.97-2.37 1.34-.58 2.97-.87 4.89-.87 1.34 0 2.65.16 3.93.48 1.28.32 2.4.78 3.36 1.38l-1.5 3.69c-1.94-1.1-3.88-1.65-5.82-1.65-1.36 0-2.37.22-3.03.66-.64.44-.96 1.02-.96 1.74s.37 1.26 1.11 1.62c.76.34 1.91.68 3.45 1.02 1.6.38 2.91.76 3.93 1.14 1.02.38 1.89.98 2.61 1.8.74.82 1.11 1.93 1.11 3.33 0 1.2-.34 2.3-1.02 3.3-.66.98-1.66 1.76-3 2.34-1.34.58-2.97.87-4.89.87zm20.24-8.61l-2.82 2.94V22h-4.83V1h4.83v9.81L60.44 1h5.4l-8.7 9.36L66.35 22h-5.67l-6.72-8.25z" fill="#05AFF2"/></svg>
```

Comment:
Hi ,

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- The background is missing
- The border of the icon is missing

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

Reply:
changes applied

Reply:
changes applied